brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post layout - Right side space.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/1b10624f-ac24-4869-bc8f-d1f9b6edf9e5?redirect=true&view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes

### Screenshots

- https://share.getcloudapp.com/DOu4oz5k

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop post layout block.
- Check is right side space is visible in editor as well as on frontend.

### Checklist:
